<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>operationalLog</class>
 <widget class="QDialog" name="operationalLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оперативный журнал</string>
  </property>
  <widget class="QPushButton" name="editBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>570</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="savebtnBox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>570</y>
     <width>241</width>
     <height>29</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="contentEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>851</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>6</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Оперативный журнал, начиная  с</string>
   </property>
  </widget>
  <widget class="QLabel" name="start">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLabel" name="content">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>510</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Содержание </string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>1001</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="action">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввод новой записи</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDate">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>25</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>7</month>
     <day>16</day>
    </datetime>
   </property>
   <property name="minimumDate">
    <date>
     <year>2020</year>
     <month>9</month>
     <day>14</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="employeeBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>480</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>570</y>
     <width>104</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>480</y>
     <width>107</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
